int32 robot_name  # robot name (follwer robot)
float32[] global_position  # global position of follower robot
---
bool register_success # registration success result by the leader